### PR TITLE
Use ES6 syntax.

### PR DESCRIPTION
**Pull request by mattcollier**
Create tabs-component here, so need to test to make sure `brTabs` is working properly.

### CODE REVIEW

File: tabs-component.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  controller: Ctrl,
+  templateUrl: 'bedrock-angular-ui/tabs-component.html'
+};
+
+/* @ngInject */
+function Ctrl($scope) {
+  var self = this;
+  self.panes = $scope.panes = [];
+  var panesMap = {};
+  panesMap.none = [];
```
Comment (dlongley) [inline tabs-component.js]: Any init should be done in `$onInit`

**Review by dlongley — CHANGES_REQUESTED** [review]

**Comment by mattcollier** [comment]
@dlongley ready for another pass.

**Review by dlongley — APPROVED** [review]

File: ui.js [hunk 2]
```diff
@@ -1,49 +1,29 @@
...
-module.directive(tooltipDirective);
-
-return module.name;
-
-});
+module.component('brHeadline', HeadlineComponent);
+module.component('brTabs', TabsComponent);
+module.directive('brActionMenu', ActionMenuDirective);
+module.directive('brError', ErrorDirective);
+module.directive('brPlaceholder', PlaceholderDirective);
```
Comment (davidlehn) [inline ui.js]: Should just be `placeholder`.  Move the related comment from end of placeholder-directive.js here.

Comment (mattcollier) [reply ui.js]: fixed.

**Review by davidlehn — APPROVED** [review]

**Pull request merged**